entrypoint/tests/__init__.py

```diff
@@ -1,3 +1,8 @@
-from entrypoint import log_conf
+import logging
 
-log_conf.init(enable_verbose=True)
+
+logger = logging.getLogger()
+logger.setLevel(logging.ERROR)
+
+handler = logging.StreamHandler()
+handler.setLevel(logging.ERROR)
```

Comment:
After adding logging when severity is not provided, a large amount of logs were output during the unit tests, so changed the log level in the test so that we can see the test result easliy.